google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -361,12 +361,12 @@ public void computeDeployments(@NotNull ComputeDeploymentsCallback callback) {
     public void disconnect() {
     }
 
-    @Nullable
+    @NotNull
     private File getFileFromFilePath(String filePath) {
```

Comment:
I think there's some changes higher up in this class where this method is invoked that also need to be changed alongside this.